hedera-node/src/main/java/com/hedera/services/txns/contract/ContractCreateTransitionLogic.java

```diff
@@ -161,6 +161,9 @@ public void doStateTransitionOperation(
 
 		// --- Do the business logic ---
 		ContractCustomizer hapiSenderCustomizer = fromHapiCreation(key, consensusTime, op);
+		if (!properties.areContractAutoAssociationsEnabled()) {
```

Comment:
Should this be reset even for ContractUpdate, in `UpdateCustomizerFactory.customizerFor`?

Reply:
Ah great point @Neeharika-Sompalli, fixing now!

Reply:
@Neeharika-Sompalli fixed by removing `MAX_AUTOMATIC_ASSOCIATIONS` from the customizer changes if disabled.

Reply:
Got it @tinker-michaelj  ! Thanks !